src/assisted_test_infra/test_infra/helper_classes/kube_helpers/agent_cluster_install.py

```diff
@@ -316,3 +316,18 @@ def _get_kubeconfig_secret() -> dict:
 
         with open(kubeconfig_path, "wt") as kubeconfig_file:
             kubeconfig_file.write(b64decode(kubeconfig_data).decode())
+
+    @classmethod
+    def list(
+        cls,
+        crd_api: CustomObjectsApi,
+        namespace,
+    ) -> List["AgentClusterInstall"]:
```

Comment:
```suggestion
    @classmethod
    def list(cls, crd_api: CustomObjectsApi, namespace) -> List["AgentClusterInstall"]:
```

Reply:
please decide on the style in all the functions in that case as i don't understand styling in that case